File: ms_cinema/domain/model/src/main/java/co/com/luisgomez29/model/movie/Movie.java
package co.com.luisgomez29.model.movie;

import co.com.luisgomez29.model.genre.Genre;
import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder(toBuilder = true)
public class Movie {
    private Integer id;
    private Genre genre;
    private String name;
    private String country;
    private Float duration;
    private String description;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: ms_cinema_kt/domain/model/src/main/kotlin/co/com/luisgomez29/model/movie/Movie.kt
package co.com.luisgomez29.model.movie

import co.com.luisgomez29.model.genre.Genre
import java.time.LocalDateTime


data class Movie(
    val id: Int,
    val genre: Genre,
    val name: String,
    val country: String,
    val duration: Float,
    val description: String,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
)